applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -589,9 +595,10 @@ static void send_report_mouse(void)
 		for (size_t i = 0; i < ARRAY_SIZE(rd->items.item); i++) {
 			struct item item = rd->items.item[i];
 
-			if (item.value) {
+			if (item.usage_id) {
 				__ASSERT_NO_MSG(item.usage_id != 0);
```

Comment:
asserting that `item.usage_id != 0` does not have much sense now

Reply:
done